### ISSUE TITLE
The container has been successfully run, but the Podman Desktop shows that it is not running, and the container cannot be started from the Podman Desktop.

### ISSUE THREAD

**Issue by edoger** [containers/podman-desktop#9830]
### Bug description

![Image](https://github.com/user-attachments/assets/a4d57351-749b-4e11-a285-c6828639ed13)
![Image](https://github.com/user-attachments/assets/c8750dec-6ef7-401a-ac47-4806e25dc4a1)

### Operating system

macOS 14.7.1  arm64

### Installation Method

Installer from website/GitHub releases

### Version

1.14.0

### Steps to reproduce

I auto-updated from v1.13.3 to v1.14.1 which seems bad, not sure what happened.

### Relevant log output

```shell

```

**Comment by cbr7** [comment]
Hi @edoger is the problem still present? Can you please give us a minimal reproducible steps to debug this issue? Also if the issue is still present can you provide the logs from the containers in question?

**Comment by edoger** [comment]
> Hi [@edoger](https://github.com/edoger) is the problem still present? Can you please give us a minimal reproducible steps to debug this issue? Also if the issue is still present can you provide the logs from the containers in question?

Hi @cbr7 

This problem still exists.
1. I have v1.13.3 version of Podman Desktop and I started the container using podman-compose and it runs fine.
2. I clicked on the Podman Desktop version in the lower right corner, and it prompted me to upgrade and asked me if I wanted to upgrade, and I chose Yes.
3. After the upgrade is complete, all containers are in a non-running state.

At this time, clicking the Run button on the container panel will have no effect and will not start the container.
When I start the container using podman-compose from the terminal, there is no change on the container panel (it used to show that the container was running).

There were no similar problems with the previous version when upgrading.

Finally, I didn't see any logs or errors!

I didn't try to reinstall, I was looking into the Podman Desktop source code to see what people had changed.

**Comment by edoger** [comment]
@cbr7 Hi, After I reinstalled the latest version, the problem still exists.

**Comment by cbr7** [comment]
@edoger Can you please clarify the steps, do you mean to say that this is an issue that happens only after you update the version? What happens if you close Podman Desktop and reopen it right now? Are the containers shown in the correct state?

**Comment by edoger** [comment]
@cbr7 The problem still exists after reopening, but now I have already completely reinstalled podman. I have not tried to automatically upgrade through the desktop in the current latest version. I am worried that the problem will occur again.

**Comment by edoger** [comment]
@cbr7 I guess the problem should be with the automatic upgrade from desktop.

**Comment by cbr7** [comment]
If the problem exists even now when simply opening PD without any upgrading why do you think the issue is with the automatic updates? Can you please provide some minimal reproducible steps? I've been trying to repro this issue without success.

**Comment by edoger** [comment]
@cbr7 Sorry, I messed up my local program a bit, and I can't reproduce the problem in a virtual machine now. The reason I guess it's happening with auto-update is because I can start the container from the command line with no problems, which means there's nothing wrong with the podman program.

**Comment by edoger** [comment]
@cbr7 If you really can't reproduce the problem, maybe it's just a problem on my machine, at least after reinstalling the installer from the dmg file, all the problems disappeared.

**Comment by odockal** [comment]
@edoger Closing as requested, feel free to open new issue if you are able to reproduce.
